corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -1,62 +1,69 @@
-hqDefine("scheduling/js/conditional_alert_main", [
-    'jquery',
-    'knockout',
-    'hqwebapp/js/initial_page_data',
-    'hqwebapp/js/widgets',
-    'data_interfaces/js/case_rule_criteria',
-    'scheduling/js/create_schedule.ko',
-    'data_interfaces/js/make_read_only',
-], function ($, ko, initialPageData) {
-    function basicInformationTab(name) {
-        var self = {};
-        self.name = ko.observable(name);
-        self.basicTabValid = ko.computed(function () {
-            return !_.isEmpty(self.name().trim());
-        });
-        self.setRuleTabVisibility = function () {
-            if (self.basicTabValid()) {
+hqDefine(
```

Comment:
It's helpful to reviewers to separate formatting-only changes from functionality changes; it's hard to tell what really changed in this commit.

Reply:
Thanks Jenny, will keep this in mind next time 👍